### ISSUE TITLE
FrankerFaceZ (ffz) расширение не работает

### ISSUE THREAD

**Issue by helenfromboston** [Flowseal/zapret-discord-youtube#360]
Перестало работать расширение ffz (FrankerFaceZ) для twitch.
В list-general добавлены строки:
cdn.betterttv.net
cdn.frankerfacez.com
*.frankerfacez.com
но не помогло. С VPN всё работает, значит, и это как-то прикрыли, получается.
Кто-нибудь знает, как возродить ffz?

**Comment by Gecker** [comment]
весь день пытаюсь возродить(через помощь других людей, не сам), не выходит.
хотя с последней версией zapret у меня даже ютуб не работает

**Comment by johnnyd-78** [comment]
Да он ни у кого, видимо, нормально не работает. Слава богу автор Быстро его своевременно откатил на предыдущую версию. Я уж подумал, у меня ссд звезда приходит или еще чего - проги запускаться перестали от слова совсем, убиваешь новый запрет и все сразу открывается что ты пытался открыть при его работе. Какой-то кривой релиз, ну его нафиг.
Лучше уж с таким "вирусом" чем с новым "не вирусом" :)

![изображение](https://github.com/user-attachments/assets/49bf5950-4330-431f-afdd-b51476d742bb)

**Comment by Gecker** [comment]
> Да он ни у кого, видимо, нормально не работает. 

о, понятно
откатился тоже.

а скрин вы приложили, это у вас заработало\вы настроили, или что?
у меня без впн смайлы FFZ не грузятся, сколько и ни мучился с изменением .bat и blacklist

**Comment by Flowseal** [comment]
> Слава богу автор Быстро его своевременно откатил на предыдущую версию.

Я ничего не откатывал

**Comment by schizak1d** [comment]
Ждем гения мысли, который сможет решить вопрос с FFZ расширением, т.к. альтернатив ему нет.

**Comment by johnnyd-78** [comment]
> Я ничего не откатывал

Я и не про вас говорил. Думаете, ваша сборка на основе запрета одна на всем свете что ли? )

**Comment by DanzoGit** [comment]
Не нужно быть гением чтобы добавить хост:
```
frankerfacez.com
```
Всё работает. И расширение и смайлы.

**Comment by johnnyd-78** [comment]
> добавить 2 хоста

Один. cdn.frankerfacez.com Это для смайлов.
Если еще и расширение куда-то там на сайт лезет - то тогда тоже один frankerfacez.com Будут работать и расширение и смайлы. Изучайте, как работают блэклисты

**Comment by DanzoGit** [comment]
> > добавить 2 хоста
> 
> Один. cdn.frankerfacez.com

Да, достаточно и этого. Но я сразу префаером и основной сайт добавляю, вдруг и его заблокируют.

> 
> Изучайте, как работают блэклисты

Ок, ок. Надеюсь планета с орбиты не сойдет, если добавлю немного лишнее.

**Comment by johnnyd-78** [comment]
> Надеюсь планета с орбит не сойдет

Не сойдет. Запрет остановится на 1-м соответствии и дальше не пойдет просто. Ситуация может двояко вылезти.

**Comment by helenfromboston** [comment]
> Не нужно быть гением чтобы добавить хост:
> 
> ```
> frankerfacez.com
> ```
> 
> Всё работает. И расширение и смайлы.

У меня не работает :(
![image](https://github.com/user-attachments/assets/13cf0351-5ad7-4440-be5d-709a37376871)

**Comment by DanzoGit** [comment]
> 
> У меня не работает :(

Ctrl+F5 пробовали?
Достаточно одного
`frankerfacez.com`

**Comment by helenfromboston** [comment]
> Ctrl+F5 пробовали?

Конечно. И перезапускать сервис. И удалять и ставить заново. И перезагружаться после каждого действия.

**Comment by DanzoGit** [comment]
> Конечно. И перезапускать сервис. И удалять и ставить заново. И перезагружаться после каждого действия.

У вас `frankerfacez.com` пингуется?

**Comment by helenfromboston** [comment]
> У вас `frankerfacez.com` пингуется?

Да
![image](https://github.com/user-attachments/assets/f406494e-0443-447d-9139-c5539f97f81d)

**Comment by DanzoGit** [comment]
@helenfromboston а если в hosts добавить?

```
104.26.6.201 frankerfacez.com
104.26.7.201 cdn.frankerfacez.com
```

**Comment by helenfromboston** [comment]
> @helenfromboston а если в hosts добавить?
> 
> ```
> 104.26.6.201 frankerfacez.com
> 104.26.7.201 cdn.frankerfacez.com
> ```

Это помогло!
Вы мой герой, спасибо!

**Comment by DanzoGit** [comment]
> Это помогло! Вы мой герой, спасибо!

Я рад) Пожалуйста)

**Comment by DanzoGit** [comment]
@helenfromboston Возможно Вам также поможет сменить dns на гугловские. Я также включил DOH. Можете загуглить, как это сделать.

**Comment by Gecker** [comment]
> @helenfromboston а если в hosts добавить?
> 
> ```
> 104.26.6.201 frankerfacez.com
> 104.26.7.201 cdn.frankerfacez.com
> ```
во-первых, спасибо
во-вторых, без понятия как это всё работает, НО:
сделал такие строки в hosts:
104.26.6.201 frankerfacez.com
104.26.6.202 cdn.frankerfacez.com
104.26.6.203 betterttv.com
104.26.6.204 cdn.betterttv.net
теперь и BTTV работает

(+ почему-то расширение FFZ не работало всё равно у меня в Firefox, поэтому заменил на скрипт запущенный через violentmonkey, как FFZ и рекомендует, и теперь всё норм)

**Comment by Flowseal** [comment]
> > Я ничего не откатывал
> 
> Я и не про вас говорил. Думаете, ваша сборка на основе запрета одна на всем свете что ли? )

А вы в каком репозитории?

**Comment by flynesss** [comment]
frankerfacez.com
betterttv.com
betterttv.net
добавил в файл List-general.txt
все работает, все смайлы и ffz и bttv и 7tv
ничего с файлом hosts не делал, вообще ничего лишнего не делал, только добавил строки

**Comment by prake1** [comment]
> 104.26.6.201 frankerfacez.com
> 104.26.6.202 cdn.frankerfacez.com
> 104.26.6.203 betterttv.com
> 104.26.6.204 cdn.betterttv.net

Спасибо, получилось заставить работать франкеры и бетертиви, добавив это в хостс, добавив строчки в сам генерал лист и сменив стратегию под моего провайдера.

**Comment by DanzoGit** [comment]
@Gecker 
> (+ почему-то расширение FFZ не работало всё равно у меня в Firefox, поэтому заменил на скрипт запущенный через violentmonkey, как FFZ и рекомендует, и теперь всё норм)

Тоже заменил на скрипт. С расширением постоянно проблемы. Очень плохо подгружается скрипт. В нем используется `cdn.frankerfacez.com`. А в скрипте `cdn2.frankerfacez.com`. Да и в целом обновы должны быстрее прилетать, нежели в расширении. Без hosts пока работает, DNS с гугловских сменил на CF.
